worker/jobs/getHierarchyJob.js

```diff
@@ -9,14 +9,14 @@
 /**
  * /worker/jobs/getHierarchyJob.js
  */
+const logger = require('winston');
```

Comment:
Wondering if you wanted to add some logging in the catch or winston was required by mistake